schema/request-schema.json

```diff
@@ -1,35 +1,39 @@
 {
-	"description": "lax ingest+publish request schema",
-	"$schema": "http://json-schema.org/draft-04/schema#",
-	"id": "https://lax.elifesciences.org/ingest-publish-request-schema.json",
-	"type": "object",
-	"additionalProperties": false,
-	"properties": {
-		"action": {
-			"enum": [ "ingest", "publish", "ingest+publish" ]
-		},
-		"location": {
-		    "description": "the location of the article to ingest/publish",
-			"type": "string",
-			"format": "uri",
-			"pattern": "^(https|file)://"
-		},
-		"id": {
-		    "description": "article identifier",
-		    "type": "string"
-		},
-		"version": {
-		    "type": "integer",
-		    "minimum": 1
-	    },
-        "force": {
-            "description": "force ingest/publish action. intended for silent corrections. use with caution.",
-          	"type": "boolean"
-        },
-		"token": {
-			"description": "data to be returned to the sender once action complete",
-			"type": "string"
-		}
-	},
-	"required": ["action", "location", "id", "version", "token"]
+    "description": "lax ingest+publish request schema",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "id": "https://lax.elifesciences.org/ingest-publish-request-schema.json",
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+    "action": {
```

Comment:
this reformatting seems to have moved to the wrong indentation the contents of `"properties"`